src/graph/rg_matrix/rg_matrix_iter.h

```diff
@@ -81,7 +80,7 @@ GrB_Info RG_MatrixTupleIter_reset
 );
 
 // free iterator data
```

Comment:
`// free iterator internals, keeping the iterator intact`
Question: are we able to reattach an iterator which we call `RG_MatrixTupleIter_attach` on ?